crates/flowistry_ide/src/mutations/find_mutations.rs

```diff
@@ -0,0 +1,39 @@
+use flowistry::{infoflow::mutation::ModularMutationVisitor, mir::aliases::Aliases};
+use log::debug;
+use rustc_hir::def_id::DefId;
+use rustc_middle::{
+  mir::{visit::Visitor, Body, Location, Place},
+  ty::TyCtxt,
+};
+
+pub fn find_mutations(
+  tcx: TyCtxt<'tcx>,
+  body: &Body<'tcx>,
+  def_id: DefId,
+  place: Place<'tcx>,
+  aliases: Aliases<'tcx>,
+) -> Vec<Location> {
+  let mut locations = vec![];
+  let pointer_aliases = aliases.reachable_values(tcx, body, def_id, place);
```

Comment:
`pointer_aliases` doesn't seem like the right name for this variable. Maybe just `reachable_values`.